backend/tarsy/services/dashboard_broadcaster.py

```
@@ -276,75 +64,43 @@
+        from tarsy.models.websocket_models import DashboardUpdate
+        
+        message = DashboardUpdate(data=data)
+        return await self.broadcast_message("dashboard_updates", message, exclude_users)
     
-    def configure_batching(self, enabled: bool, batch_size: int = 5, timeout_seconds: int = 2):
-        """Configure message batching settings."""
-        self.batching_enabled = enabled
-        self.batch_size = batch_size
-        self.batch_timeout_seconds = timeout_seconds
-        logger.info(f"Batching configured: enabled={enabled}, size={batch_size}, timeout={timeout_seconds}s") 
+    async def broadcast_session_update(self, session_id: str, data: Dict[str, Any], exclude_users: Set[str] = None) -> int:
```

Ruff (0.12.2)

Line too long (123 > 88)

(E501)